include/ImNodeFlow.h

```diff
@@ -904,12 +905,14 @@ namespace ImFlow
          * @param style Style of the pin
          */
         explicit Pin(PinUID uid, std::string name, ConnectionFilter filter, PinType kind, BaseNode* parent, ImNodeFlow** inf, std::shared_ptr<PinStyle> style)
-            :m_uid(uid), m_name(std::move(name)), m_filter(filter), m_type(kind), m_parent(parent), m_inf(inf), m_style(std::move(style))
+            :m_uid(uid), m_name(std::move(name)), m_type(kind), m_filter(filter), m_style(std::move(style)), m_parent(parent), m_inf(inf)
```

Comment:
Same here, I prefer the order in which they appear in the constructor parameters. so if it gives a warning, it's somewhere else where the order needs changing

Reply:
Ok, I will revert those changes and instead change the order of the declared variables.